homeassistant/components/sabnzbd/sensor.py

```diff
@@ -23,13 +23,14 @@ class SabnzbdSensor(SensorEntity):
 
     def __init__(self, sensor_type, sabnzbd_api_data, client_name):
         """Initialize the sensor."""
-        self._client_name = client_name
-        self._field_name = SENSOR_TYPES[sensor_type][2]
-        self._name = SENSOR_TYPES[sensor_type][0]
+        meta_data = SENSOR_TYPES[sensor_type]
```

Comment:
Pass in the sensor description into the entity, and set it as `self.entity_description`

Reply:
Done